src/dotnet/ForwardingApp.cs

```diff
@@ -1,28 +1,14 @@
-﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
+﻿using Microsoft.DotNet.Cli.Utils;
+using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using Microsoft.DotNet.Cli.Utils;
+using System.Text;
 
 namespace Microsoft.DotNet.Cli
 {
-    /// <summary>
-    /// A class which encapsulates logic needed to forward arguments from the current process to another process
-    /// invoked with the dotnet.exe host.
-    /// </summary>
     public class ForwardingApp
```

Comment:
Why not just use the ForwaringAppImplementation? And call that one ForwardingApp?

If you are worried about the fact that this class is public, don't, we don't ship the CLI as a library.